x/go/telem/telem.go

```diff
@@ -437,7 +437,9 @@ const (
 // AlignmentPair is essentially two array index values that can be used to represent
 // the location of a sample within a group of arrays. For example, if you have two arrays
 // that have 50 elements each, and you want the 15th element of the second array, you would
-// use NewAlignmentPair(1, 15).
+// use NewAlignmentPair(1, 15). The first index is called the 'domain index' and the second
+// index is called the 'sample index'. The domain index is the index of the array, and the
```

Comment:
This isn't super clear to me. So you're saying the domain index is what you use to access an individual array from a set of arrays?